### ISSUE TITLE
Package Request

### ISSUE THREAD

**Issue by maheese** [testorg-2020/cflinuxfs3-CRAN#50]
Package: geiger

**Comment by maheese** [comment]
There was an issue processing your package addition request. Tagging @maheese to debug and closing the issue. See the logs:
```
Error in value[[3L]](cond): package 'phangorn' is not available (for R version 4.0.2)

**Comment by maheese** [comment]
There was an issue processing your package addition request. Tagging @maheese to debug and closing the issue. See the logs:
```
Error in value[[3L]](cond): package 'phangorn' is not available (for R version 4.0.2)
